src/views/PulseMessages/index.jsx

```diff
@@ -362,25 +367,28 @@ export default class PulseMessages extends Component {
               headers={['Exchange', 'Routing Key']}
             />
           </List>
+          {listening ? (
+            <Button
+              requiresAuth
+              variant="round"
+              spanProps={{ className: classes.startStopIconSpan }}
+              tooltipProps={{ title: 'Stop Listening' }}
+              onClick={this.handleStopListening}
+              className={classes.stopIcon}>
+              <StopIcon />
+            </Button>
+          ) : (
+            <Button
+              requiresAuth
+              variant="round"
+              spanProps={{ className: classes.startStopIconSpan }}
+              tooltipProps={{ title: 'Start Listening' }}
+              onClick={this.handleStartListening}
+              className={classes.playIcon}>
```

Comment:
"Start Listening" is missing the following prop:

`disabled={!bindings.length}`